apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -1,11 +1,140 @@
+$small: 850px;
+
+.btn {
+  color: white;
+  text-transform: uppercase;
+  text-align: center;
+  width: 100%;
+  background-color: #D32E6A;
+  padding: 0.5em;
+  cursor: pointer;
+}
+
+.close-btn {
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  z-index: 200;
+  width: 31px;
+  height: 31px;
+  cursor: pointer;
+}
+
+.reactions-container {
+  padding: 20px;
+  display: flex;
+  flex-flow: column;
+  margin: -5px;
+  position: absolute;
+  top: 65px;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 90%;
+  max-width: 1000px;
+
+  @media (min-width: $small) {
+    flex-flow: row wrap;
+  }
+}
+
+.main-column {
+  width: 100%;
+  margin: 5px;
+
+  @media (min-width: $small) {
+    width: calc(67% - 10px);
+  }
+}
+
+.side-column {
+  width: 100%;
+  margin: 5px;
+
+  img {
+    max-width: 100%;
+  }
+
+  @media (min-width: $small) {
+    width: calc(33% - 10px);
+    order: 1;
+  }
+}
+
 .reactions {
+  .reactions-modal {
+    display: none;
+  }
+
+  &.open {
+    .reactions-modal {
+      display: block;
+      z-index: 10000;
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+
+      &:before {
+        position: fixed;
+        display: block;
+        content: ' ';
+        width: 100%;
+        height: 100%;
+        background-color: #f8f8f8;
+        opacity: 0.95;
```

Comment:
`0.95` should be written without a leading zero as `.95`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)